include/seastar/http/client.hh

```diff
@@ -76,6 +76,7 @@ class connection : public enable_shared_from_this<connection> {
     hook_t _hook;
     future<> _closed;
     internal::client_ref _ref;
+    bool _keep = true;
```

Comment:
how about using the terminology in the HTTP/1.1 standard, say, s/_keep/_persistent/?